prime-router/src/test/kotlin/ReceiverTests.kt

```diff
@@ -12,6 +13,43 @@ import kotlin.test.Test
 internal class ReceiverTests {
     private val translatorConfig = mockkClass(TranslatorConfiguration::class)
 
+    @Test
+    fun `test condition filter on non full-elr pipeline`() {
+        val receiver = Receiver(
+            name = "elr",
+            organizationName = "IGNORE",
+            topic = Topic.COVID_19,
+            customerStatus = CustomerStatus.INACTIVE,
+            conditionFilter = listOf("blah"),
+            translation = translatorConfig,
+            externalName = "Ignore ELR"
+        )
+
+        assertThat(
+            receiver.consistencyErrorMessage(
+                UnitTestUtils.simpleMetadata
+            )
+        ).isEqualTo(
+            "Invalid value for condition filter. " +
+                "Cannot have a condition filter in pipelines other than universal pipeline."
```

Comment:
(optional) The approach to error message checking in tests seems to vary and be kind of a judgment call, but you might want to consider changing this to use `messageContains` or `isNotNull` to make this less dependent on the specific error message text.

Reply:
I can't get messageContains to work for some reason, and I don't feel like isNotNull is enough since it should be returning a different error than the one checked for so I think I am going to stick with what I have. It would be really nice if we used a different test library that had better support for things like this.